### PR TITLE
Allow `plugins_page_skipped` parameter in Onboarding API endpoint

### LINKED ISSUE #38659: Core Profiler skip plugins page not saving onboarding options correctly
> woocommerce_onboarding_profile option should have plugins_page_skipped set to true

This test failed for me. Not sure what went wrong, I had no errors after skipping. My option values after the onboarding:

```
{
  "business_choice": "im_just_starting_my_business",
  "selling_online_answer": null,
  "selling_platforms": null,
  "is_store_country_set": true,
  "industry": [
    null
  ],
  "completed": true
}
```

_Originally posted by @ilyasfoo in https://github.com/woocommerce/woocommerce/pull/38577#pullrequestreview-1469448785_

### PR DESCRIPTION
**Pull request by ilyasfoo**
### Changes proposed in this Pull Request:

Closes #38659.

### How to test the changes in this Pull Request:

1. Install and activate WooCommerce on a brand new site
1. Install the WooCommerce Beta Tester plugin
1. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper
1. Enable the 'core-profiler' feature flag
1. Visit your public URL /wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard
4. Go through the new core profiler flow until the plugins step
5. Skip the plugins step by clicking on `Skip this step`
6. Go to `Tools > WCA Test Helper`
7. In the Options tab, search for `woocommerce_onboarding_profile`
8. Click on `Edit`
9. Observe the value `plugins_page_skipped`: `true` exists

### CODE REVIEW

File: plugins/woocommerce/src/Admin/API/OnboardingProfile.php [hunk 1]
```diff
@@ -449,6 +449,13 @@ public static function get_profile_properties() {
 				'readonly'          => true,
 				'validate_callback' => 'rest_validate_request_arg',
 			),
+			'plugins_page_skipped' => array(
```
Comment (adrianduffell) [inline plugins/woocommerce/src/Admin/API/OnboardingProfile.php]:
```suggestion
			'is_plugins_page_skipped' => array(
```
Nit: the other boolean params have an `is_` prefix

**Comment by adrianduffell** [comment]
@ilyasfoo Looks like there's a test that needs an updated assertion:

```
1) WC_Admin_Tests_API_Onboarding_Profiles::test_schema
Failed asserting that actual size 18 matches expected size 17.

/var/www/html/wp-content/plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/onboarding-profile.php:134
```

**Review by adrianduffell — APPROVED** [review]
Tested well for me and looks good! Thanks @ilyasfoo 🚀 

I left a nit and also it looks like a minor change is needed for a test to pass

File: plugins/woocommerce/src/Admin/API/OnboardingProfile.php (hunk 1 above)
Comment (ilyasfoo) [reply plugins/woocommerce/src/Admin/API/OnboardingProfile.php]: Good call! Renamed it in [2486c2b](https://github.com/woocommerce/woocommerce/pull/38764/commits/2486c2b9e6fb2bdf68fdc280d209b3e95979faec)

**Comment by ilyasfoo** [comment]
Thanks, Adrian! Merging since I'm confident with 1 review.

**Pull request merged**